### PR TITLE
Filter transform link_teaser

### LINKED ISSUE #3306: Filter & Transform paragraph-link_teaser content ## Problem Statement
Our tome-sync data doesn't match up with our templates, and we need to follow the steps [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/2835) to ensure that the data will be compatible with our current templates.

## Goal
Content from `paragraph-link_teaser` entities match their corresponding entities from `pages.json`

### PR DESCRIPTION
**Pull request by bkjohnson**
### Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/3306

Adds filters and transformers for link_teaser content

## Testing done
- `npm run test:unit ./src/site/stages/build/process-cms-exports/tests/`
- `npm run build:content:test -- -n paragraph.c5fec275-9ce2-480c-af68-a6ad65e9ff65.json`

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/schemas/raw/paragraph-link_teaser.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { getFilter } = require('../../filters');
+
+/* eslint-disable camelcase */
+module.exports = {
+  type: 'object',
+  properties: {
+    field_link: {
+      type: 'array',
+      maxItems: 1,
```
Comment (cvalarida) [inline src/site/stages/build/process-cms-exports/schemas/raw/paragraph-link_teaser.js]: How confident are you that this will always be only one item long?

**Review by cvalarida — APPROVED** [review]
I think to fix that failing unit test, you should be able to merge in the latest master and it'll be golden.

File: src/site/stages/build/process-cms-exports/schemas/raw/paragraph-link_teaser.js (hunk 1 above)
Comment (bkjohnson) [reply src/site/stages/build/process-cms-exports/schemas/raw/paragraph-link_teaser.js]: _Reasonably_ confident?  It seemed that a `link_teaser` was a singular thing based on how the transformed object looked:

```json
  "entity": {
    "fieldLink": {
      "url": {
        "path": "https://www.benefits.va.gov/gibill/my_story.asp"
      },
      "title": "My Story—How Veterans and Servicemembers Use the GI Bill",
      "options": []
    },
    "fieldLinkSummary": "Hear inspiring stories from Veterans and Servicemembers who’ve advanced their education and training with the Post-9/11 GI Bill."
  }
```

**Comment by bkjohnson** [comment]
@cvalarida [This](https://github.com/department-of-veterans-affairs/vets-website/pull/11213/commits/3fa5f0e3cf7f1d6157e3f5cbbc5c61b660be0873) is the only real change since your approval.

**Pull request merged**